rasa/core/channels/slack.py

```diff
@@ -180,16 +180,12 @@ def _is_user_message(slack_event):
     @staticmethod
     def _sanitize_user_message(text, uids_to_remove):
         """Remove superfluous/wrong/problematic tokens from a message.
-
```

Comment:
Could you also add these empty lines back?

Reply:
hope this is fine now :)